trade-finance-manager-ui/server/graphql/queries/deal-query.js

```diff
@@ -1,8 +1,8 @@
 const gql = require('graphql-tag');
 
 const dealQuery = gql`
-  query Deal($_id: String! $tasksFilters: TasksFilters) {
-    deal(params: { _id: $_id, tasksFilters: $tasksFilters }) {
+  query Deal($_id: String! $tasksFilters: TasksFilters $activityFilters: ActivityFilters) {
+    deal(params: { _id: $_id, tasksFilters: $tasksFilters activityFilters: $activityFilters }) {
```

Comment:
Can we update the API test to have all the new things? in `graphql-query-tests/get-deal.api-test.js`

Reply:
done